src/Orleans.Core.Abstractions/Placement/HashBasedPlacement.cs

```diff
@@ -6,7 +6,7 @@ namespace Orleans.Runtime
     public  class HashBasedPlacement : PlacementStrategy
     {
 
-        internal static HashBasedPlacement Singleton { get; } = new HashBasedPlacement();
+        public static HashBasedPlacement Singleton { get; } = new HashBasedPlacement();
```

Comment:
No need to make all these internal methods/properties/classes public if Abstractions have InternalsVisibleTo for Orleans, OrleansRuntime, and TesterInternal, which we need anyway.